feat(cli): Add `--binary <path>` option to `expo run` commands for reusing builds. (#30437)
expo/expo: docs/pages/more/expo-cli.mdx

- `--no-bundler`: Skip starting the dev server. Enabled automatically if the dev server is already serving the app from a different process.
- `-d, --device [device]`: Device name or ID to build the app on. You can pass `--device` without arguments to select a device from a list of available options. This supports connected devices as well as virtual devices.
- `-p, --port <port>`: Port to start the development server. **Default: 8081**. This is only relevant for development builds. Production builds will [export](#exporting) the project and embed the files in the native binary before installing them on a device.

> - `--binary <path>`: File path to the binary to install on the device. When this is provided, the build process will be skipped and the binary will attempt to be installed directly. If the binary was not built for the correct device, e.g. built for simulator, installed on device, then the command will fail.

Reviewer: ```suggestion
- `--binary <path>`: File path to the binary to install on the device. When this is provided, the build process will be skipped and the binary will attempt to be installed directly. If the binary was not built for the correct device, for example, it is built for the simulator or installed on the device, then the command will fail.
```

Just a tiny suggestion to avoid Latin abbreviations as per our writing style guide and improve the readability of the example scenario here.
